src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const array = sourceString.split(';');
+
+  const object = array.reduce((accumulator, current) => {
```

Comment:
[CODE STYLE] The same applies here, avoid variable namings like `object`. A more descriptive name like `styleObject` or `cssStyles` would improve readability.